test/expected/plan_expand_hypertable_optimized.out

```diff
@@ -1085,3 +1085,122 @@ SELECT * FROM cte ORDER BY value;
+                  QUERY PLAN                  
+----------------------------------------------
+ Sort
+   Sort Key: h.value
+   ->  Append
+         ->  Seq Scan on _hyper_1_1_chunk h
+         ->  Seq Scan on _hyper_1_2_chunk h_1
+         ->  Seq Scan on _hyper_1_3_chunk h_2
+(6 rows)
+
+:PREFIX SELECT * FROM hyper WHERE chunks_in(hyper, ARRAY[1,2,3]) AND time < 10 ORDER BY value;
+                QUERY PLAN                
+------------------------------------------
+ Sort
+   Sort Key: _hyper_1_1_chunk.value
+   ->  Append
+         ->  Seq Scan on _hyper_1_1_chunk
+               Filter: ("time" < 10)
+         ->  Seq Scan on _hyper_1_2_chunk
+               Filter: ("time" < 10)
+         ->  Seq Scan on _hyper_1_3_chunk
+               Filter: ("time" < 10)
+(9 rows)
+
+:PREFIX SELECT * FROM hyper_ts h JOIN tag on (h.tag_id = tag.id ) WHERE chunks_in(h, ARRAY[116]) AND time < to_timestamp(10) AND device_id = 'dev1' ORDER BY value;
+                                                               QUERY PLAN                                                               
+----------------------------------------------------------------------------------------------------------------------------------------
+ Sort
+   Sort Key: h.value
+   ->  Hash Join
+         Hash Cond: (tag.id = h.tag_id)
+         ->  Seq Scan on tag
+         ->  Hash
+               ->  Append
+                     ->  Seq Scan on _hyper_3_116_chunk h
+                           Filter: (("time" < 'Wed Dec 31 16:00:10 1969 PST'::timestamp with time zone) AND (device_id = 'dev1'::text))
+(9 rows)
+
+:PREFIX SELECT * FROM hyper_w_space h1 JOIN hyper_ts h2 ON h1.device_id=h2.device_id WHERE chunks_in(h1, ARRAY[104,105]) AND chunks_in(h2, ARRAY[116,117]) ORDER BY h1.value;
+                         QUERY PLAN                          
+-------------------------------------------------------------
+ Sort
+   Sort Key: h1.value
+   ->  Hash Join
+         Hash Cond: (h2.device_id = h1.device_id)
+         ->  Append
+               ->  Seq Scan on _hyper_3_116_chunk h2
+               ->  Seq Scan on _hyper_3_117_chunk h2_1
+         ->  Hash
+               ->  Append
+                     ->  Seq Scan on _hyper_2_104_chunk h1
+                     ->  Seq Scan on _hyper_2_105_chunk h1_1
+(11 rows)
+
+:PREFIX SELECT * FROM hyper h1, hyper h2 WHERE chunks_in(h1, ARRAY[1,2]) AND chunks_in(h2, ARRAY[2,3]);
+                     QUERY PLAN                      
+-----------------------------------------------------
+ Nested Loop
+   ->  Append
+         ->  Seq Scan on _hyper_1_1_chunk h1
+         ->  Seq Scan on _hyper_1_2_chunk h1_1
+   ->  Materialize
+         ->  Append
+               ->  Seq Scan on _hyper_1_2_chunk h2
+               ->  Seq Scan on _hyper_1_3_chunk h2_1
+(8 rows)
+
+SET enable_seqscan=false;
+:PREFIX SELECT time FROM hyper WHERE time=0 AND chunks_in(hyper, ARRAY[1]);
```

Comment:
add a comment that this should be an INDEX ONLY scan and the reason we are worried about that.

Reply:
added